storage/storage.go

```diff
@@ -105,41 +106,48 @@ type Client struct {
 func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error) {
 	var host, readHost, scheme string
 
+	// In general, it is recommended to use NewService instead of NewClient, since NewService
```

Comment:
Any tests to update? Should there be?

Reply:
Yes, I'm looking to add unit test coverage for the matrix in the doc I linked.

Reply:
Added tests.